wallet/wallet.go

```diff
@@ -3667,6 +3672,7 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 	// node that already has it in their mempool.
 	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L590
 	case match(err, "txn-already-in-mempool"):
+		log.Infof("%v: tx already in mempool", txid)
```

Comment:
Should these be warning or info?

Reply:
Given that the original `Removed invalid transaction` is info, I went with info as well. Warn would probably lead to too many questions from concerned users. Normally this isn't cause for concern as it could just be a sweep attempt that was later replaced with a higher fee transaction.